client/my-sites/customer-home/free-photo-library-card/index.jsx

```diff
@@ -84,4 +90,6 @@ const FreePhotoLibraryCard = ( { recordTracksEvent: tracks } ) => {
 	);
 };
 
-export default connect( null, { recordTracksEvent } )( FreePhotoLibraryCard );
+export default connect( null, { openSupportArticleDialog, recordTracksEvent } )(
```

Comment:
Nonblocking, but since `openSupportArticleDialog` is a redux action, we can decorate the meaningful action with some meta that says to also fire an analytics event as a middleware side-effect. (This gets added to the redux action as a `meta.analytics` property.

So we could do something like:

```jsx
import {
	withAnalytics,
	recordTracksEvent,
} from 'state/analytics/actions';
//...

<Button onClick={ openSupportAndTrack }>{ translate( 'Learn more' ) }</Button>
//...
const openSupportAndTrack = () => {
    withAnalytics(
		recordTracksEvent( 'calypso_customer_home_free_photo_library_video_support_page_viewr' ),
		openSupportArticleDialog( 145498, 'https://support.wordpress.com/free-photo-library/' )
	);
}

export default connect( null, { openSupportAndTrack, recordTracksEvent } )( FreePhotoLibraryCard );
```

So if we inspect redux state, while clicking on the button, we'll see that only one action (openSupportArticleDialog)  is dispatched, but that it also has an extra `meta.analytics` property.

Let me know if that didn't make any sense 👍

Reply:
Thanks @gwwar! I've added a commit that I think addresses this. :)